examples/multisig/basic.eg.ts

```diff
@@ -81,7 +81,7 @@ $.assert($.array($.sizedUint8Array(32)), approvals)
 
 // Approve the proposal as Carol (final approval).
 await multisig
```

Comment:
Should we get rid of this intermediate non-executing ratify? Would simplify the example.